diff --git a/home/roman/backup/home-manager/home-manager/config.nix b/home/roman/backup/home-manager/home-manager/config.nix
deleted file mode 100644
index 19daeb5..0000000
--- a/home/roman/backup/home-manager/home-manager/config.nix
+++ /dev/null
@@ -1,19 +0,0 @@
-# Here lives shit I don't understand
-{pkgs, config, ... }:
-
-{
-  home.username = "roman";
-  home.homeDirectory = "/home/roman";
-  home.stateVersion = "23.11"; # Do not change
-
-  # Enable XDG desktop portal
-  xdg.portal = {
-    enable = true;
-    extraPortals = [ pkgs.xdg-desktop-portal-gtk ];
-    config.common.default = "gtk";
-  };
-
-  # Enable Pipewire
-
-  programs.home-manager.enable = true;
-}
diff --git a/home/roman/backup/home-manager/home-manager/flake.lock b/home/roman/backup/home-manager/home-manager/flake.lock
deleted file mode 100644
index f909396..0000000
--- a/home/roman/backup/home-manager/home-manager/flake.lock
+++ /dev/null
@@ -1,83 +0,0 @@
-{
-  "nodes": {
-    "ags": {
-      "inputs": {
-        "nixpkgs": "nixpkgs"
-      },
-      "locked": {
-        "lastModified": 1708986959,
-        "narHash": "sha256-pTh3n2Lp8Twr8Qv30E86gU6NfoNoId7Da+6P1qEG/Jc=",
-        "owner": "Aylur",
-        "repo": "ags",
-        "rev": "ee3e65e8003a39c2ff1f46dca60308771da6072c",
-        "type": "github"
-      },
-      "original": {
-        "owner": "Aylur",
-        "repo": "ags",
-        "type": "github"
-      }
-    },
-    "home-manager": {
-      "inputs": {
-        "nixpkgs": [
-          "nixpkgs"
-        ]
-      },
-      "locked": {
-        "lastModified": 1708451036,
-        "narHash": "sha256-tgZ38NummEdnXvxj4D0StHBzXgceAw8CptytHljH790=",
-        "owner": "nix-community",
-        "repo": "home-manager",
-        "rev": "517601b37c6d495274454f63c5a483c8e3ca6be1",
-        "type": "github"
-      },
-      "original": {
-        "owner": "nix-community",
-        "repo": "home-manager",
-        "type": "github"
-      }
-    },
-    "nixpkgs": {
-      "locked": {
-        "lastModified": 1708475490,
-        "narHash": "sha256-g1v0TsWBQPX97ziznfJdWhgMyMGtoBFs102xSYO4syU=",
-        "owner": "NixOS",
-        "repo": "nixpkgs",
-        "rev": "0e74ca98a74bc7270d28838369593635a5db3260",
-        "type": "github"
-      },
-      "original": {
-        "owner": "NixOS",
-        "ref": "nixos-unstable",
-        "repo": "nixpkgs",
-        "type": "github"
-      }
-    },
-    "nixpkgs_2": {
-      "locked": {
-        "lastModified": 1708296515,
-        "narHash": "sha256-FyF489fYNAUy7b6dkYV6rGPyzp+4tThhr80KNAaF/yY=",
-        "owner": "nixos",
-        "repo": "nixpkgs",
-        "rev": "b98a4e1746acceb92c509bc496ef3d0e5ad8d4aa",
-        "type": "github"
-      },
-      "original": {
-        "owner": "nixos",
-        "ref": "nixos-unstable",
-        "repo": "nixpkgs",
-        "type": "github"
-      }
-    },
-    "root": {
-      "inputs": {
-        "ags": "ags",
-        "home-manager": "home-manager",
-        "nixpkgs": "nixpkgs_2"
-      }
-    }
-  },
-  "root": "root",
-  "version": 7
-}
diff --git a/home/roman/backup/home-manager/home-manager/flake.nix b/home/roman/backup/home-manager/home-manager/flake.nix
deleted file mode 100644
index 855dfa7..0000000
--- a/home/roman/backup/home-manager/home-manager/flake.nix
+++ /dev/null
@@ -1,35 +0,0 @@
-{
-  description = "Home Manager configuration of roman";
-
-  inputs = {
-    # Specify the source of Home Manager and Nixpkgs.
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-    home-manager = {
-      url = "github:nix-community/home-manager";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
-
-    # AGS
-    ags.url = "github:Aylur/ags";
-  };
-
-  outputs = { nixpkgs, home-manager, ... }@inputs:
-    let
-      system = "x86_64-linux";
-      pkgs = nixpkgs.legacyPackages.${system};
-    in {
-      homeConfigurations."roman" = home-manager.lib.homeManagerConfiguration {
-        inherit pkgs;
-
-        # Specify your home configuration modules here, for example,
-        # the path to your home.nix.
-        modules = [ ./home.nix ];
-
-        # Optionally use extraSpecialArgs
-        # to pass through arguments to home.nix
-
-	# AGS
-	extraSpecialArgs = { inherit inputs; };
-      };
-    };
-}
diff --git a/home/roman/backup/home-manager/home-manager/home.nix b/home/roman/backup/home-manager/home-manager/home.nix
deleted file mode 100644
index f869802..0000000
--- a/home/roman/backup/home-manager/home-manager/home.nix
+++ /dev/null
@@ -1,40 +0,0 @@
-#  _   _    ____    _    _   _____
-# | | | |  / __ \  | \  / | |  ___|
-# | |_| | | |  | | |  \/  | | |__
-# |  _  | | |  | | | |\/| | |  __|
-# | | | | | |__| | | |  | | | |___
-# |_| |_|  \____/  |_|  |_| |_____|
-
-# By Roman Bezroutchko
-
-
-{ inputs, config, pkgs, ... }:
-
-{
-  imports = [
-    ./package_config/package_config.nix
-    ./config.nix
-  ];
-  nixpkgs.config.allowUnfree = true;
-
-  home.packages = with pkgs; [
-    # Essentials
-    floorp
-    kitty
-    (waybar.overrideAttrs (oldAttrs: { mesonFlags = oldAttrs.mesonFlags ++ [ "-Dexperimental=true" ]; }))
-    mako
-    libnotify
-    hyprpaper
-    wofi
-    nerdfonts
-
-    # Other
-    lf
-    vscodium 
-    lutris
-    libreoffice
-  ];
-
-  home.file = {
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/ags/ags.nix b/home/roman/backup/home-manager/home-manager/package_config/ags/ags.nix
deleted file mode 100644
index f3ae57e..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/ags/ags.nix
+++ /dev/null
@@ -1,31 +0,0 @@
-#             _____   _____
-#     /\     / ____| / ____|
-#    /  \   | |  __  | (___  
-#   / /\ \  | | |_ | \___  \ 
-#  / ____ \ | |__| |  ____) |
-# /_/    \_\ \_____| |_____/ 
-                        
-# By Roman Bezroutchko
-
-{ inputs, config, pkgs, ... }:
-{
-  # add the home manager module
-  imports = [ 
-    inputs.ags.homeManagerModules.default 
-  ];
-
-  programs.ags = {
-    enable = true;
-
-    # additional packages to add to gjs's runtime
-    extraPackages = with pkgs; [
-      gtksourceview
-      webkitgtk
-      accountsservice
-    ];
-  };
-
-  xdg.configFile = {
-    "ags/config.js".source = ./config.js;
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/ags/bar.js b/home/roman/backup/home-manager/home-manager/package_config/ags/bar.js
deleted file mode 100644
index e69de29..0000000
diff --git a/home/roman/backup/home-manager/home-manager/package_config/ags/config.js b/home/roman/backup/home-manager/home-manager/package_config/ags/config.js
deleted file mode 100644
index ce08770..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/ags/config.js
+++ /dev/null
@@ -1,5 +0,0 @@
-export default {
-    windows: [
-        // this is where window definitions will go
-    ]
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/ags/variables.js b/home/roman/backup/home-manager/home-manager/package_config/ags/variables.js
deleted file mode 100644
index e69de29..0000000
diff --git a/home/roman/backup/home-manager/home-manager/package_config/hypr/appearance.nix b/home/roman/backup/home-manager/home-manager/package_config/hypr/appearance.nix
deleted file mode 100644
index 637d4a5..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/hypr/appearance.nix
+++ /dev/null
@@ -1,60 +0,0 @@
-{ config, pkgs, ... }:
-
-{
-  wayland.windowManager.hyprland.settings = {
-    general = {
-      gaps_in = 5;
-      gaps_out = 20;
-      border_size = 2;
-      "col.active_border" = "rgba(33ccffee) rgba(00ff99ee) 45deg";
-      "col.inactive_border" = "rgba(595959aa)";
-
-      layout = "dwindle";
-
-      allow_tearing = false;
-    };
-
-    decoration = {
-      rounding = 10;
-
-      blur = {
-        enabled = true;
-        size = 3;
-        passes = 1;
-        
-        vibrancy = 0.1696;
-      };
-
-      drop_shadow = true;
-      shadow_range = 4;
-      shadow_render_power = 3;
-      "col.shadow" = "rgba(1a1a1aee)";
-    };
-
-    animations = {
-      enabled = true;
-      bezier = [ "myBezier, 0.05, 0.9, 0.1, 1.05" ];
-      animation = [ 
-	"windows, 1, 7, myBezier"
-    	"windowsOut, 1, 7, default, popin 80%"
-    	"border, 1, 10, default"
-    	"borderangle, 1, 8, default"
- 	"fade, 1, 7, default"
-    	"workspaces, 1, 6, default"
-      ];
-    };
-
-    dwindle = {
-      pseudotile = true;
-      preserve_split = true;
-    };
-
-    master = {
-      new_is_master = true;
-    };
-
-    misc = {
-      force_default_wallpaper = 1;
-    };
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/hypr/hyprland.nix b/home/roman/backup/home-manager/home-manager/package_config/hypr/hyprland.nix
deleted file mode 100644
index 234a376..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/hypr/hyprland.nix
+++ /dev/null
@@ -1,31 +0,0 @@
-#  _    ___     _______  _____  _               _   _ _____  
-# | |  | \ \   / /  __ \|  __ \| |        /\   | \ | |  __ \ 
-# | |__| |\ \_/ /| |__) | |__) | |       /  \  |  \| | |  | |
-# |  __  | \   / |  ___/|  _  /| |      / /\ \ | . ` | |  | |
-# | |  | |  | |  | |    | | \ \| |____ / ____ \| |\  | |__| |
-# |_|  |_|  |_|  |_|    |_|  \_\______/_/    \_\_| \_|_____/ 
-
-# By Roman Bezroutchko
-
-{ config, pkgs, ... }:
-
-{
-  imports = [
-    # Startup
-    ./startup.nix
-
-    # Appearance
-    ./appearance.nix
-
-    # Input
-    ./input.nix
-
-    # Keybindings
-    ./keybinds.nix
-
-    # Other
-    ./other.nix
-  ];  
-
-  wayland.windowManager.hyprland.enable = true;
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/hypr/input.nix b/home/roman/backup/home-manager/home-manager/package_config/hypr/input.nix
deleted file mode 100644
index 61594b0..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/hypr/input.nix
+++ /dev/null
@@ -1,23 +0,0 @@
-{ config, pkgs, ... }:
-
-{
-  wayland.windowManager.hyprland.settings = {
-    input = {
-      kb_layout = "us";
-      kb_variant = "";
-      kb_model ="";
-      kb_options ="";
-      kb_rules ="";
-      follow_mouse = 1;
-
-      touchpad = {
-        natural_scroll = true;
-      };
-      sensitivity = 0;
-    };
-
-    gestures = {
-      workspace_swipe = true;
-    };
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/hypr/keybinds.nix b/home/roman/backup/home-manager/home-manager/package_config/hypr/keybinds.nix
deleted file mode 100644
index 7b2adfa..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/hypr/keybinds.nix
+++ /dev/null
@@ -1,41 +0,0 @@
-{ config, pkgs, ... }:
-
-let
-  # Workspace bindings
-  workspaces_num = [ 1 2 3 4 5 6 7 8 9 ];
-  generateWorkspace = num: "SUPER, ${toString num}, workspace, ${toString num}";
-  generateMoveToWorkspace = num: "SUPER SHIFT, ${toString num}, movetoworkspace, ${toString num}";
-  workspaceBindings = (map generateWorkspace(workspaces_num)) ++ (map generateMoveToWorkspace(workspaces_num));
-
-in 
-{
-  wayland.windowManager.hyprland.settings = {
-    bind = [
-      # Controls
-      "SUPER, Space, killactive"
-
-      "SUPER, H, movefocus, l"
-      "SUPER SHIFT, H, movewindow, l"
-      "SUPER, J, movefocus, d"
-      "SUPER SHIFT, J, movewindow, d"
-      "SUPER, K, movefocus, u"
-      "SUPER SHIFT, K, movewindow, u"
-      "SUPER, L, movefocus, r"
-      "SUPER SHIFT, L, movewindow, r"
-
-      # Important applications
-      "SUPER, Tab, exec, wofi --show drun"
-      "SUPER, Return, exec, kitty"
-      "SUPER, B, exec, floorp"
-
-      # Other applications
-    ] 
-    ++
-    workspaceBindings;
-
-    bindm = [
-      "SUPER, mouse:272, movewindow"
-      "SUPER, mouse:273, resizewindow"
-    ];
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/hypr/other.nix b/home/roman/backup/home-manager/home-manager/package_config/hypr/other.nix
deleted file mode 100644
index baa7aa3..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/hypr/other.nix
+++ /dev/null
@@ -1,9 +0,0 @@
-{ config, pkgs, ... }:
-
-{
-  wayland.windowManager.hyprland.settings = {
-    monitor = [
-      ", preferred, auto, 1"
-    ];
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/hypr/plugins.nix b/home/roman/backup/home-manager/home-manager/package_config/hypr/plugins.nix
deleted file mode 100644
index e69de29..0000000
diff --git a/home/roman/backup/home-manager/home-manager/package_config/hypr/startup.nix b/home/roman/backup/home-manager/home-manager/package_config/hypr/startup.nix
deleted file mode 100644
index 32c0717..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/hypr/startup.nix
+++ /dev/null
@@ -1,14 +0,0 @@
-{ config, pkgs, ... }:
-
-{
-  wayland.windowManager.hyprland.settings = {
-    env = [
-      "XCURSOR_SIZE, 24"
-      "QT_QPA_PLATFORMTHEME, qt5ct"
-    ];
-
-    exec-once = [
-      "waybar"
-    ];
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/other.nix b/home/roman/backup/home-manager/home-manager/package_config/other.nix
deleted file mode 100644
index 38ba247..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/other.nix
+++ /dev/null
@@ -1,19 +0,0 @@
-{ config, pkgs, ... }:
-
-{
-  # Git
-  programs.git = {
-    enable = true;
-    userName = "Jozhin-s-Bazhin";
-    userEmail = "rbezroutchko@gmail.com";
-  };
-
-  # GTK
-  gtk = {
-    enable = true;
-    font = {
-      name = "Sans";
-      size = 11;
-    };
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/package_config.nix b/home/roman/backup/home-manager/home-manager/package_config/package_config.nix
deleted file mode 100644
index 2004f66..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/package_config.nix
+++ /dev/null
@@ -1,10 +0,0 @@
-{ pkgs, config, ... }:
-
-{
-  imports = [
-    ./other.nix
-    ./ags/ags.nix
-    ./hypr/hyprland.nix
-    ./zsh.nix
-  ];
-}
diff --git a/home/roman/backup/home-manager/home-manager/package_config/zsh.nix b/home/roman/backup/home-manager/home-manager/package_config/zsh.nix
deleted file mode 100644
index d357641..0000000
--- a/home/roman/backup/home-manager/home-manager/package_config/zsh.nix
+++ /dev/null
@@ -1,24 +0,0 @@
-{ config, pkgs, ... }:
-
-{ 
-  programs.zsh = {
-    enable = true;
-    enableAutosuggestions = true;
-    enableCompletion = true;
-    enableVteIntegration = true;
-
-    # Aliases
-    shellAliases = {
-
-      # Basic stuff
-      v = "nvim";
-      vi = "nvim";
-      vim = "nvim";
-
-      # Nix rebuilds
-      rbs = "bash ~/scripts/rebuild-aliases/rebuild-switch";
-      rbh = "bash ~/scripts/rebuild-aliases/home-switch";
-      rba = "bash ~/scripts/rebuild-aliases/backup-all";
-    };
-  };
-}
diff --git a/home/roman/backup/home-manager/home-manager/style.nix b/home/roman/backup/home-manager/home-manager/style.nix
deleted file mode 100644
index e69de29..0000000
diff --git a/home/roman/backup/home-manager/home-manager/weird_shit.nix b/home/roman/backup/home-manager/home-manager/weird_shit.nix
deleted file mode 100644
index 35d6715..0000000
--- a/home/roman/backup/home-manager/home-manager/weird_shit.nix
+++ /dev/null
@@ -1,4 +0,0 @@
-{ config, pkgs, ... }:
-
-{
-  
diff --git a/home/roman/backup/home-manager/package_config/zsh.nix b/home/roman/.config/home-manager/package_config/zsh.nix
index d357641..c171fc3 100644
--- a/home/roman/backup/home-manager/package_config/zsh.nix
+++ b/home/roman/.config/home-manager/package_config/zsh.nix
@@ -18,7 +18,7 @@
       # Nix rebuilds
       rbs = "bash ~/scripts/rebuild-aliases/rebuild-switch";
       rbh = "bash ~/scripts/rebuild-aliases/home-switch";
-      rba = "bash ~/scripts/rebuild-aliases/backup-all";
+      rba = "bash ~/scripts/rebuild-aliases/backup";
     };
   };
 }
